gdal-src/build.rs

```diff
@@ -0,0 +1,58 @@
+fn main() {
+    let sqlite3_include_dir =
+        std::env::var("DEP_SQLITE3_INCLUDE").expect("This is set by libsqlite3-sys");
+    let sqlite3_lib_dir = std::env::var("DEP_SQLITE3_LIB_DIR").expect("set by libsqlite3-sys");
+    let proj_root = PathBuf::from(std::env::var("DEP_PROJ_ROOT").expect("set by proj-sys"));
+    let proj_lib = if proj_root.join("lib").join("proj_d.lib").exists() {
+        "proj_d.lib"
+    } else if proj_root.join("lib").join("proj.lib").exists() {
+        "proj.lib"
+    } else {
+        "libproj.a"
+    };
+
+    let res = cmake::Config::new("source")
```

Comment:
For anyone wondering, `cmake` [automatically sets](https://docs.rs/cmake/latest/cmake/struct.Config.html#method.profile) `CMAKE_BUILD_TYPE`.